.github/workflows/release.yml

```diff
@@ -106,6 +106,7 @@ jobs:
         with:
           generate_release_notes: true
           tag_name: v${{ env.PKG_VERSION }}
+          body_path: .github/release.yml
```

Comment:
Why did we add this?

Reply:
Looking into this more I don't think we need this. Previously I thought it was a link to specify where the release notes are, but while reading the documentation more it states "If body is specified, the body will be pre-pended to the automatically generated notes". I don't think that we need a path to the release notes configuration and that by setting the body path it would just add the text before the auto-generated notes.

Reply:
Good learning! Please do remove it!